rocky/rocky/views/finding_list.py

```diff
@@ -94,6 +104,15 @@ def get_queryset_params(self):
             "asc_desc": self.sorting_order,
         }
 
+    def get_table_columns(self) -> dict[str, str]:
+        return {
+            "severity": _("Severity"),
+            "finding": _("Finding"),
+            "location": _("Location"),
+            "tree": _("Tree"),
+            "graph": _("Graph"),
+        }
```

Comment:
Can be a static variable `TABLE_COLUMNS` outside the class

(also the type isn't correct, the `_` function doesn't return a `str`)